tech-specs/src/explore/design/pos.md

```diff
@@ -0,0 +1,258 @@
+# Proof of Stake (PoS) system
+
+## Epoch
+
+An epoch is a range of blocks, whose length is set by the `epoch_duration` [parameter](#system-parameters). Epochs are identified by consecutive integers starting at 0. For each epoch, we set the [relevant data](#epoched-data) in advance by the `pipeline_length` [parameter](#system-parameters).
```

Comment:
It could also be a range in time (timestamp), epochs just end at a particular (non-decreasing) parameter value

Reply:
I see. We may want to specify and implement epochs in the base ledger as they will be used for DKG (and maybe elsewhere) too, right?

Reply:
Yes, definitely

Reply:
Thanks! I've updated this section to reflect that and removed the `epoch_duration` parameter. I've opened #331, let's discuss it in the ledger team meeting.